org.deluge_torrent.deluge.yaml

```diff
@@ -0,0 +1,437 @@
+app-id: org.deluge_torrent.deluge
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: deluge-gtk
+#rename-icon: deluge
+#copy-icon: true
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=xdg-download
+
+modules:
+
+- name: boost
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://dl.bintray.com/boostorg/release/1.75.0/source/boost_1_75_0.tar.bz2
+    sha256: 953db31e016db7bb207f11432bef7df100516eeb746843fa0486a222e3fd49cb
+  build-commands:
+  - ./bootstrap.sh --prefix="${FLATPAK_DEST}" --with-python=python3 
+  - ./b2 stage  -j $FLATPAK_BUILDER_N_JOBS variant=release threading=multi link=shared debug-symbols=off
+  - ./b2 install --prefix="${FLATPAK_DEST}" variant=release threading=multi link=shared debug-symbols=off
+
+- name: libtorrent
+  buildsystem: cmake-ninja
+  builddir: true
+  config-opts:
+    - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    - -Dpython-bindings=ON 
+    - -Dboost-python-module-name=python
+  sources:
+    - type: archive
+      url: https://github.com/arvidn/libtorrent/releases/download/v1.2.12/libtorrent-rasterbar-1.2.12.tar.gz
+      sha256: c3744ac9fa41f6e6ebf79538a2ea678df76a2cbbaf3ac6ae2c05455314e5cce8
+
+## Python3 modules
```

Comment:
FYI you can use https://github.com/flatpak/flatpak-builder-tools/tree/master/pip to generate python dependencies.

Reply:
Is it OK, if I don't want to use flatpak-builder-tools' pip generator?

Reply:
Yeah, whatever works for you. Just posted it so you know it exists.